studying.md

```diff
@@ -183,17 +183,25 @@ mainly left around for compatibility with older Anki versions.
 button on the [deck overview](studying.md#study-overview) screen.) This is useful if
 you cannot answer the card at the moment or you want to come back to it
 another time. Burying can also [happen automatically](studying.md#siblings-and-burying) for
-cards of the same note. If cards were in learning when they are buried,
+cards of the same note. 
+
+With the old scheduler, if cards were in learning when they are buried,
```

Comment:
I noticed another section in the manual that kept info for both the old and new schedulers. Do we want to continue like this? Or focus just on relevant info for the latest version now that everyone has been prompted to update? 

If the later, then I can also update the documentation to remove references to the old verison.

Reply:
It will likely be quite a while until everyone has updated, as some users don't upgrade immediately, even if an update notification is sent out. So unfortunately we're probably best off mentioning both old and new behaviour for now.

Because people will often Google the part they're interested in, we can't rely on them to have read in chronological order, so it might also be worth turning one mention of the scheduler into a hyperlink in each section, eg 

With the [new scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html) however, burying cards ..

Otherwise looks good - thank you!

Reply:
Thanks, makes sense! I have just now added in the links, so it should be good to go.

Reply:
Thanks, that makes sense! I just added in the links, so it should be good to go now.